### PR TITLE
[Doc] Update `Crypto` example

### PR DESCRIPTION
**Pull request by DarkiStar**
Removed redundant new() initializations

### CODE REVIEW

**Comment by timothyqiu** [comment]
I think the `extends Node` and `_ready()` setup is also unrelated information. The example could be just ... code.

**Review by AThousandShips — DISMISSED** [review]
Some issues:
* The indentation must be four spaces, you have changed it to tabulators
* You've moved the member variables into the `_Ready` method in C#, that's invalid C# AFAIK, they shouldn't have `private`, or leading underscore
* And I agree with the above, it should just be a simpler example

File: doc/classes/Crypto.xml [hunk 1]
```diff
@@ -1,75 +1,76 @@
...
+        # Generate new RSA key.
+        var key = crypto.generate_rsa(4096)
+        
+        # Generate new self-signed certificate with the given key.
+        var cert = crypto.generate_self_signed_certificate(key, "CN=mydomain.com,O=My Game Company,C=IT")
+        
+        # Save key and certificate in the user folder.
+        key.save("user://generated.key")
+        cert.save("user://generated.crt")
+
```
Comment (AThousandShips) [inline doc/classes/Crypto.xml:22]:
```suggestion

```

**Comment by AThousandShips** [comment]
It should be tabs for everything except code indentation, as it was originally

**Comment by DarkiStar** [comment]
Github Editor is kind of difficult ...
I will make another try. Tabs for everything except code. But it looks strange ...

**Comment by AThousandShips** [comment]
The first two indentations for the code should be tabs, like it was originally 

I'd suggest using an editor on a computer, you're going to have to squash your commits anyway before this can be merged and it's much easier than the web editor

File: doc/classes/Crypto.xml [hunk 2]
```diff
@@ -5,69 +5,71 @@
 	</brief_description>
 	<description>
 		The Crypto class provides access to advanced cryptographic functionalities.
-		Currently, this includes asymmetric key encryption/decryption, signing/verification, and generating cryptographically secure random bytes, RSA keys, HMAC digests, and self-signed [X509Certificate]s.
+		Currently, this includes asymmetric key encryption/decryption, signing/verification, and generating
+		cryptographically secure random bytes, RSA keys, HMAC digests, and self-signed [X509Certificate]s.
```
Comment (AThousandShips) [inline doc/classes/Crypto.xml:9]: This line shouldn't be broken up into two

**Review by AThousandShips — APPROVED** [review]
Makes sense and looks good to me

Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by DarkiStar** [comment]
Allready commited my crypto branch to my godot master fork. So there are no differences between my crypto branch and master. Not sure how to continue.

**Comment by AThousandShips** [comment]
You should reset your master branch to the upstream branch and then rebase it, or rebase against the upstream branch

**Comment by DarkiStar** [comment]
Thank you for your patience. Can I somehow trigger the rebase/reset of the master branch directly on GitHub, or does there necessarily have to be a copy on my computer to initiate these functions?

**Comment by AThousandShips** [comment]
You need to do this on a computer, you can't squash in the web interface

**Comment by DarkiStar** [comment]
I would like to manage it on my own.

**Pull request closed without merging**